src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -68,28 +67,28 @@ export const HandbookWrapper = styled.div`
         background-color:${(props) => props.theme.secondaryLightColor};
         color: white;
     }
-    
+
     .page-header-section {
-        height: 10rem;
-        text-align: center;
-        background: rgb(71,126,150);
-        background: linear-gradient(250deg, rgba(71,126,150,1) 0%, rgba(0,179,159,1) 35%, rgba(60,73,79,1) 100%);
-        h1 {
-            line-height: 10rem;
-            color: white;
-        }
+      height: 10rem;
+      text-align: center;
+      background: rgb(71,126,150);
+      background: linear-gradient(250deg, rgba(71,126,150,1) 0%, rgba(0,179,159,1) 35%, rgba(60,73,79,1) 100%);
+      h1 {
+          line-height: 10rem;
+          color: white;
+      }
```

Comment:
These formatting changes are not required, I think you are using any different formatter like "Prettier" or something which is causing these changes, kindly revert these formatting changes for this file !! 

Let's wait for @YashKamboj and @debo19 views on it I am not sure about it !! , I think we are using ESlint

Reply:
Yeah! these changes make difficult to review